awx/main/management/commands/callback_stats.py

```diff
@@ -0,0 +1,38 @@
+import time
+import sys
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        super(Command, self).__init__()
```

Comment:
is there any reason that `handle` calls the super `__init__`?

Reply:
Nope, I think that's just a copy-paste typo.